test/test_02_03.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from hm_1.tasks.task_02_03 import combinations
+
+
+@pytest.fixture()
```

Comment:
Just inline these fixtures, it will be more readable to see them inside the tests.

Reply:
I didn’t understand  what means „inline fixure “, can you explain it more detail?

Reply:
Remove this fixture and move its content to the tests

```python
def test_combinations_with_2_lists_with_not_equal_sizes():
    expected_result = [[1, 3], [2, 3]]
    assert combinations([1, 2], [3]) == expected_result
```